Cargo.toml

```diff
@@ -55,10 +55,12 @@ serde_bytes = "0.11.5"
 
 [dev-dependencies]
 assert_matches = "1.2"
-serde_bytes = "0.11"
+criterion = "0.3.0"
 pretty_assertions = "0.6.1"
-chrono = { version = "0.4", features = ["serde"] }
 uuid = { version = "0.8.1", features = ["serde", "v4"] }
+proptest = "0.10"
```

Comment:
looks like this crate is now at 1.0.0 -- should we bump the dependency?

Reply:
Good catch, fixed.